docs/marketplace/dev/feature-walkthroughs/202108.0/marketplace-product-offer-prices-feature-walkthrough.md

```diff
@@ -13,8 +13,48 @@ To learn more about the feature and to find out how end users use it, see [Produ
 
 {% endinfo_block %}
 
-## Related Developer articles
+## Marketplace Product Offer Prices - Overview
+Product offer prices can be defined for each product offer by merchants. 
+If price has not defined for product offer, will be used price from concrete product which is tied to the product offer by default.
+Each price should have type. 
+The price type it is an entity to define what type of prices will be used for selling. 
+There are 2 price types defined in the Spryker system, it's ORIGINAL and DEFAULT. 
```

Comment:
https://documentation.spryker.com/docs/prices-overview Can be a text from this document be reused? Looks clearer

Reply:
I've added link to this document, below.